.github/workflows/CREATE_MILESTONE.yml

```diff
@@ -12,32 +12,35 @@ jobs:
     steps:
       - name: Get new Milestone Title
         id: getTitle
-        env: 
+        env:
           TITLE: ${{github.event.milestone.title}}
         run: |
           MILESTONE_NUMBER=${TITLE:1}
           INCREMENTED_NUMBER=$((MILESTONE_NUMBER + 1))
           echo "MILESTONE_NAME=M$INCREMENTED_NUMBER" >> $GITHUB_OUTPUT
+      - name: Check if Milestone already exists
+        id: checkIfMilestoneExists
```

Comment:
Consider our discussions on steps naming in https://github.com/camunda/camunda-modeler/pull/4650 adjust your changes accordingly.